exec/init_expr.go

```diff
@@ -36,9 +36,10 @@ func execInitExpr(expr []byte, globals []int64) int64 {
 
 	for {
 		b, err := r.ReadByte()
-		if err == io.EOF {
+		switch err {
+		case io.EOF:
 			break
```

Comment:
This break applies to the switch and not to the for loop there is a similar case below in the same function with:
```
		case ops.End:
			break
```